src/mask.js

```diff
@@ -178,7 +179,7 @@ angular.module('ui.mask', [])
                                 }
                                 iElement.bind('blur', blurHandler);
                                 iElement.bind('mousedown mouseup', mouseDownUpHandler);
-                                iElement.bind('input keyup click focus', eventHandler);
+                                iElement.bind(listOptions.eventsToHandle.join(' '), eventHandler);
```

Comment:
Actually I meant it to be ```linkOptions``` but I typoed... thanks for letting me know what was failing the build. I hadn't taken the time to investigate that yet.